openassessment/xblock/test/test_submission.py

```diff
@@ -22,6 +22,7 @@
 from submissions.api import SubmissionInternalError, SubmissionRequestError
 
 from .base import XBlockHandlerTestCase, scenario
+from mock import patch
```

Comment:
Could you group imports, which relates to the mock package?

Reply:
Done